src/Cli/dotnet/Commands/Run/VirtualProjectBuildingCommand.cs

```diff
@@ -1180,11 +1180,16 @@ public static void WriteProjectFile(
             // Note that ArtifactsPath needs to be specified before Sdk.props
             // (usually it's recommended to specify it in Directory.Build.props
             // but importing Sdk.props manually afterwards also works).
+            //
+            // An empty value is always given for BaseIntermediateOutputPath and BaseOutputPath, to ensure that
+            // the SDK targets will initialize them based on ArtifactsPath, and not use values for those paths coming in from Directory.Build.props.
             writer.WriteLine($"""
                 <Project>
 
                   <PropertyGroup>
                     <IncludeProjectNameInArtifactsPaths>false</IncludeProjectNameInArtifactsPaths>
+                    <BaseIntermediateOutputPath></BaseIntermediateOutputPath>
+                    <BaseOutputPath></BaseOutputPath>
```

Comment:
I think this won't work (and the failing test seems to agree with me). Implicit build files are imported through SDKs below (see where we emit `<Import`) so that will overwrite any values here.

For properties that cannot be overwritten, see the block where we write `<RestoreUseStaticGraphEvaluation>false</RestoreUseStaticGraphEvaluation>` for example.